indico/modules/events/abstracts/models/comments.py

```diff
@@ -17,9 +17,19 @@
 from __future__ import unicode_literals
 
 from indico.core.db import db
-from indico.core.db.sqlalchemy import UTCDateTime
+from indico.core.db.sqlalchemy import PyIntEnum, UTCDateTime
 from indico.util.date_time import now_utc
 from indico.util.string import format_repr, return_ascii, text_to_repr
+from indico.util.struct.enum import IndicoEnum
+
+
+class AbstractCommentVisibility(int, IndicoEnum):
```

Comment:
Right now there are 2 use cases that are pretty clear:
 * Messages between reviewers and conveners/judges (e.g. "This abstract sucks", which shouldn't be seen by other reviewers as those can even be authors of the abstract);
 * Messages between the conveners and the managers (e.g. "The abstract sucks, but Ome's review of it sucks even more. We shouldn't have made him a reviewer.");